shared/utils/logUtils.go

```diff
@@ -17,6 +18,55 @@ import (
 	"gopkg.in/natefinch/lumberjack.v2"
 )
 
+var redact_regex = regexp.MustCompile(`(password\s+)[^\s"]+`)
```

Comment:
```suggestion
var redactRegex = regexp.MustCompile(`(password\s+)[^\s"]+`)
```

I'm not sure I've seen any pythonic variable name in Go code for now.